core/src/main/mima-filters/2.0.1.backwards.excludes

```diff
@@ -0,0 +1,6 @@
+# Internal API
```

Comment:
We released 2.0.2 so please update this filename accordingly.

Reply:
fixed